pages/Product/index.jsx

```diff
@@ -11,7 +11,7 @@ import ProductContent from './components/Content';
  */
 const Product = ({ id }) => (
   <View>
-    {id && <ProductContent />}
+    {id && <ProductContent id={id} />}
```

Comment:
Please remove the id check here. the sub-components should take care about showing or not by themselves.